### PR TITLE
Add the missing keybindings in ruby-on-rails layer

### PR DESCRIPTION
**Pull request by asok**
@sdwolfz those commands were missing in the spacemacs key map.

### CODE REVIEW

File: layers/+frameworks/ruby-on-rails/packages.el [hunk 1]
```diff
@@ -44,6 +44,8 @@
         "ffv" 'projectile-rails-find-view
         "ffy" 'projectile-rails-find-layout
         "ff@" 'projectile-rails-find-mailer
+        "ffw" 'projectile-rails-find-webpack
+        "ffb" 'projectile-rails-find-job
```
Comment (sdwolfz) [inline layers/+frameworks/ruby-on-rails/packages.el]: This should be ordered alphabetically under `"ffa" 'projectile-rails-find-locale`.

Comment (asok) [reply layers/+frameworks/ruby-on-rails/packages.el]: Done.

Comment (sdwolfz) [reply layers/+frameworks/ruby-on-rails/packages.el]: You should do it in the documentation as well.

Comment (asok) [reply layers/+frameworks/ruby-on-rails/packages.el]: Double done.

**Comment by sdwolfz** [comment]
Thank you :heart:!
Cherry-picked into develop branch, you can safely delete your branch.

**Pull request closed without merging**